config-templates/salesforce/leads.yaml

```diff
@@ -0,0 +1,71 @@
+data_source:
+  name: salesforce_lead
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.lead ### Update to your leads schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 10 * * *
+  identifiers:
+    - name: lead_id
+      type: primary
+      expr: id
+    - name: account_id
+      type: foreign
+      expr: converted_account_id
+    - name: opportunity_id
+      type: foreign
+      expr: converted_opportunity_id
+    - name: contact_id
+      type: foreign
+      expr: converted_contact_id
+    - name: owner_id
+      type: foreign
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: true
+        time_granularity: day
+      expr: created_date
+    - name: converted_date
+      type: time
+      type_params:
+        is_primary: false
+        time_granularity: day 
+    - name: is_deleted
+      type: categorical
+    - name: name
+      type: categorical
+    - name: company
+      type: categorical
+    - name: lead_source
+      type: categorical
+    - name: is_converted
+      type: categorical
+  measures:
```

Comment:
Wow, measures after dimensions. Very slick... I like this more generally. It's probably closer to the way people organize their tables: id, dimensions, measures.

Reply:
I don't have strong opinions here! I like to keep identifiers and dimensions next to each other cause i think of them as more similar.